### DISCUSSION TITLE
Admin: General.editmenuExternalnamehint is not displayed after change

### DISCUSSION THREAD

**Discussion by Nemecida** [requarks/wiki#6000] [Error / Bug Report] [answered]
Display Admin: General.editmenuExternalnamehint in the Admin block: General.editShortCuts after changing with github, we are not displayed correctively.
![зображення](https://user-images.githubusercontent.com/121578517/209864572-151329a6-94e7-482d-acaf-400106adaf43.png)
![зображення](https://user-images.githubusercontent.com/121578517/209864739-9d7bd362-3f76-4749-9a93-4d4dc0d1f48b.png)

**Comment by NGPixel** [comment, marked as answer]
Make sure your instance is connected to the internet so that it can download the localization files.

If that's not possible, you can sideload the localization files manually:
https://docs.requarks.io/install/sideload

**Comment by Nemecida** [comment]
Hi, so the language package is set. Instance is connected to the internet. Ukrainian, Polish, English. The result is not important from the language. Wiki.js version 2.5.295
![зображення](https://user-images.githubusercontent.com/121578517/209931286-688d9b3e-b42b-401b-aea8-4190ab4a0f50.png)

**Reply by NGPixel** [reply]
Have you cleared your browser cache? Try in another browser.

**Reply by Nemecida** [reply]
Used 3 browsers: Chrome, Opera GX, Firefox. The cache cleaned everywhere. Similarly on my mobile phone.
